stylesheets/components/cards.css

```diff
@@ -155,4 +160,45 @@ margin-right: 1rem;
 }
 .card-container-vertical .card-title{
     margin-top:1rem;
+}
+/* horizontal card */
+.card-container-horizontal{
+/* width:21rem; */
+width:20rem;
+}
+.card-container-horizontal .card{
+
+}
+.card-container-horizontal .card-section{
+    flex-wrap: wrap;
+}
+.card-container-horizontal .card-img-container{
+    width:10rem;
+    height: 6rem;
+}
+.card-container-horizontal img{
+    /* height: 100%; */
+    width: 100%;
+    /* height: 100%; */
+}
+.card-container-horizontal .card-actions{
+margin-top: 0;
+margin-left:1rem;
+margin-right:1rem;
+margin-bottom: 1rem;
```

Comment:
you can also used shorhand property of margin(0 1 1 1) to  avoid multi- line

Reply:
changes done